index.html

```diff
@@ -35,38 +31,151 @@ <h1 class="greeting-hearder">I’m Sive <br> Glad to see you!</h1>
+        <div class="card-container">
+            <img src="img/Snapshoot Portfolio.png" alt="Project Snapshoot" class="project-img">
+            <h2 class="card-tittle">Tonic</h2>
+            <div class="frame2-img">
+                <img src="img/Frame 2.png" alt="">
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+
+        <div class="card-container">
+            <img src="img/Snapshoot Portfolio(1).png" alt="Project Snapshoot" class="project-img">
+            <h2 class="card-tittle">Multi-Post Stories</h2>
+            <div class="frame2-img">
+                <img src="img/Frame 2.png" alt="">
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+
+        <div class="card-container">
+            <img src="img/Snapshoot Portfolio(2).png" alt="Project Snapshoot" class="project-img">
+            <h2 class="card-tittle">Tonic</h2>
+            <div class="frame2-img">
+                <img src="img/Frame 2.png" alt="">
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+
+        <div class="card-container">
+            <img src="img/Snapshoot Portfolio(3).png" alt="Project Snapshoot" class="project-img">
+            <h2 class="card-tittle">Multi-Post Stories</h2>
+            <div class="frame2-img">
+                <img src="img/Frame 2.png" alt="">
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+    </section>
+    
+    <section class="about">
+        <h1 class="about-h1">About <br> Myself</h1>
```

Comment:
- to match the Figma design kindly :

- [ ] add some margin on top of your about me the title
- [ ] Add the missing top right border-radius 

![image](https://user-images.githubusercontent.com/89282221/221351138-6226b94e-ec51-48ef-bb80-4cba53626e3d.png)